apps/extension/src/template.dashboard.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="./favicon.svg" />
+    <link rel="icon" href="./favicon.svg?v=<%= __webpack_hash__  %>" />
```

Comment:
Browsers aggressively cache favicons, this hash is needed in order to get it to update between dev and canary builds (so when the extension id doesn't change).